google/cloud/storage/_media/requests/upload.py

```diff
@@ -79,9 +79,7 @@ def retriable_request():
 
             return result
 
-        return _request_helpers.wait_and_retry(
-            retriable_request, self._get_status_code, self._retry_strategy
-        )
+        return _request_helpers.wait_and_retry(retriable_request, self._retry_strategy)
 
 
```

Comment:
nit: to match download, we probably want to add the retry arg docstrings for MultipartUpload, ResumableUpload etc

Reply:
Thanks, done